tests/testsuite/shell_completions.rs

```diff
@@ -0,0 +1,56 @@
+use completest_pty::Runtime;
+
+fn complete(input: &str, shell: &str) -> String {
+    let shell = shell.into();
+
+    // Load the runtime
+    let mut runtime = match shell {
+        "bash" => load_runtime::<completest_pty::BashRuntimeBuilder>("bash"),
+        "elvish" => load_runtime::<completest_pty::ElvishRuntimeBuilder>("elvish"),
+        "fish" => load_runtime::<completest_pty::FishRuntimeBuilder>("fish"),
+        "zsh" => load_runtime::<completest_pty::ZshRuntimeBuilder>("zsh"),
+        _ => panic!("Unsupported shell: {}", shell),
+    };
```

Comment:
Any code using `completest_pty` needs a `#[cfg(unix)]`

Reply:
I saw it in the CI, thanks for your reminder!